packages/ckeditor5-clipboard/src/dragdrop.ts

```diff
@@ -244,6 +244,13 @@ export default class DragDrop extends Plugin {
 		this.listenTo<ViewDocumentDragStartEvent>( viewDocument, 'dragstart', ( evt, data ) => {
 			const selection = modelDocument.selection;
 
+			// Don't drag from non-editable place.
+			if ( !model.isSelectableEditable( selection ) ) {
+				data.preventDefault();
+
+				return;
+			}
```

Comment:
Is there any specific reason to disable dragging when the root is readonly? AFAIR, it is not disabled currently.